locale/en/LC_MESSAGES/django.po

```diff
@@ -894,11 +1299,11 @@ msgstr "There are no course instances for this course."
 
 #: course/views.py
 msgid "ENROLLMENT_ERROR_CANNOT_ENROLL_OR_ALREADY_ENROLLED"
-msgstr "You cannot enroll, or have already enrolled, in this course."
+msgstr "You cannot enrol, or have already enrolled, in this course."
```

Comment:
heh :) This string should be edited to include more concrete status update, but that's most likely requires some development

Reply:
This particular this string aside, is the plan to adopt/use British English in A+? I’d vote against that, since almost all the English learning materials served on A+ are (presumably) in American English, and this would just create an unattractive contrast between the platform and (most of) the contents.

Reply:
At least I understood that that was the plan, since the materials and names at Aalto in general are in British English (e.g. "Learning Centre"), and the language code had been defined as en-gb. However, if most of the learning materials are in face in American English, that would definitely be a good reason to stick with American English.

Reply:
I don’t know this for a fact, but my gut feeling is that AmE is way more common in learning materials. And yeah, aligning with learning materials on the same site seems more important than aligning with generic Aalto policies that have little impact on the grassroots level of teaching and learning.